### ISSUE TITLE
Allow 1x1 tile size for Build Health Details

### ISSUE THREAD

**Issue by bajmming** [ALM-Rangers/Visualize-Team-Project-Health-Widgets#11]

**Comment by jakobehn** [comment]
What information would you like to see when using this tile size? The available area is pretty small in this case

**Comment by bajmming** [comment]
Sorry for delay...I didn't receive (or missed) the email alert.  We have a dashboard at the moment which shows the status of 40 (and increasing) database repos.  All we really need is the repo name and whether it's successful, in progress or failed, all the rest of the info is useful, but not required.  If the repo name or any text got chopped off due to the 1x1 size it wouldn't matter as if everything's green we don't care and we can click on the red (failed) one's and find out more information.

**Comment by WouterDeKort** [comment]
@bajmming Sounds good! I'll put it on the backlog. If you want, you could also submit a pull request. I would be happy to help you with that.

**Comment by bajmming** [comment]
@WouterDeKort yeah I'd be happy to try a pull request :)  I've had a quick look and looks like it would involve mainly updating the manifest suportedSizes and the css.
